apolloschurchapp/ios/apolloschurchapp/apolloschurchapp.entitlements

```diff
@@ -4,8 +4,6 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
-	<key>com.apple.developer.networking.wifi-info</key>
```

Comment:
may still need this for Airplay, I'm not sure...

Reply:
I think this is just a cast thing